editor/test/integration/subselection_test.clj

```diff
@@ -133,8 +133,8 @@
 
 (g/defnode MoveManip
   (input selection g/Any)
-  (output position g/Any (g/fnk [selection _basis]
-                                (let [evaluation-context (g/make-evaluation-context {:basis _basis})
+  (output position g/Any (g/fnk [selection]
+                                (let [evaluation-context (g/make-evaluation-context)
```

Comment:
Whoa. Super-weird that we're evaluating `g/node-value` inside an output. We better hope that no-one caches this downstream, or we'll run into trouble. 😨

Reply:
Yeah, that's wild. Good thing it's a test and not a production code I guess...